File: C#/fibonacci.cs
using System;
public class Class {
    public static void Main() {
        ulong a = 0uL;
        Console.WriteLine(a);
        ulong b = 1uL;
        Console.WriteLine(b);
        ulong c = a + b;
        
        do {
            Console.WriteLine(c);
            a = b;
            b = c;
            c = a + b;
        } while (c >= b);

        Console.WriteLine("Overflowed!");
    }
}

File: Kotlin/fibonacci.kt
fun main(args: Array<String>) {
    var a: ULong = 0u;
    println(a);
    var b: ULong = 1u;
    println(b);
    var c: ULong = a + b;

    do {
        println(c);
        a = b;
        b = c;
        c = a + b;
    } while (c >= b);

    println("Overflowed!");
}
